app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -26,6 +28,18 @@ def show
 
 private
 
+  def set_path_set_id_param
+    return true if anonymous_feedback_params[:path_set_id].presence
+
+    paths = anonymous_feedback_params[:paths].presence
```

Comment:
```suggestion
    paths = anonymous_feedback_params[:paths].presence
    return unless paths
```

Then we don't need the `if` below and it follows the pattern of the line above.

Reply:
👍